### PR TITLE
Clean up plugin doco

### PR DESCRIPTION
**Pull request by mikeminutillo**
Replacement for #2580

### CODE REVIEW

File: servicecontrol/plugins/index.md [hunk 1]
```diff
@@ -20,16 +20,35 @@ ServiceControl provides the monitoring capability by analyzing the configured er
 To allow ServiceControl to monitor endpoints:
 
  1. ServiceControl should be [installed](/servicecontrol/installation.md) and at least one instance should be configured using the same transport as that of the endpoints that are being monitored.
- 1. For every endpoint that is being monitored by ServiceControl, [configure the endpoint for auditing](/nservicebus/operations/auditing.md#configuring-auditing). Make sure that the audit queue is the same as the audit queue that ServiceControl is configured with.
- 1. For every endpoint that is being monitored by ServiceControl, [configure recoverability](/nservicebus/recoverability/). Make sure that the error queue is the same as the error queue that ServiceControl is configured with.
+ 2. For every endpoint that is being monitored by ServiceControl, [configure the endpoint for auditing](/nservicebus/operations/auditing.md#configuring-auditing). Make sure that the audit queue is the same as the audit queue that ServiceControl is configured with.
```
Comment (SimonCropp) [inline servicecontrol/plugins/index.md]: `ing). Make s`  shouldnt that be a comma?

Comment (mikeminutillo) [inline servicecontrol/plugins/index.md]: > For every endpoint that is being monitored by ServiceControl, configure the endpoint for auditing. Make sure that the audit queue is the same as the audit queue that ServiceControl is configured with.

vs

> For every endpoint that is being monitored by ServiceControl, configure the endpoint for auditing, make sure that the audit queue is the same as the audit queue that ServiceControl is configured with.

With the comma I think it needs an "and"

> For every endpoint that is being monitored by ServiceControl, configure the endpoint for auditing, and make sure that the audit queue is the same as the audit queue that ServiceControl is configured with.

Comment (SimonCropp) [inline servicecontrol/plugins/index.md]: > With the comma I think it needs an "and"

agreed. and the point below has the same issue

**Comment by weralabaj** [comment]
@mikeminutillo seems there are some issues with partials versions, the component name is customchecks not customchecks3, customchecks4, etc. so the file names should be customchecks_3, customchecks_4.

As for V6 versions 1 and 2, I think we should keep them on a single page.

**Comment by mikeminutillo** [comment]
@weralabaj I thought this identifier was the nuget alias which is:

```
CustomChecks3: ServiceControl.Plugin.Nsb3.CustomChecks
CustomChecks4: ServiceControl.Plugin.Nsb4.CustomChecks
CustomChecks5: ServiceControl.Plugin.Nsb5.CustomChecks
CustomChecks6: ServiceControl.Plugin.Nsb6.CustomChecks
```

I'll try adding explicit version info for each one.

**Comment by weralabaj** [comment]
@mikeminutillo I'm not 100% sure myself ;) But component keys/aliases are listed here https://github.com/Particular/docs.particular.net/blob/master/components/components.yaml.

**Comment by mikeminutillo** [comment]
@weralabaj This whole section is overly complicated. Each plugin has a separate repo and nuget package for each major of core. None of our other repos/packages do that. The doco engine is not the only point of friction with this model. The whole thing is kind of insane.

**Comment by mikeminutillo** [comment]
It works! Backing away slowly and holding breath

**Comment by mikeminutillo** [comment]
@Particular/docs-maintainers I am happy with this if you want to merge it.

**Pull request merged**